.travis.yml

```diff
@@ -101,10 +110,10 @@ script:
 - if [[ $SKIP_MAKE != y && $RUN_INTEGRATION_TESTS != y ]]; then make -C tests run_local_quick ; fi
 - if [[ $SKIP_MAKE != y ]]; then make install ; fi
 - if [[ -z $NO_ARTIFACTS ]]; then (cd dest && tar cvzf ../artifacts/librdkafka-${CC}.tar.gz .) ; fi
-- for distro in $ADDITIONAL_BUILDS ; do packaging/tools/distro-build.sh $distro || exit 1 ; done
+- if [[ "$TRAVIS_ARCH" != "s390x" ]]; then for distro in $ADDITIONAL_BUILDS ; do packaging/tools/distro-build.sh $distro || exit 1 ; done; fi
```

Comment:
Don't believe this is needed since ADDITIONAL_BUILDS is not defined for the s390 worker

Reply:
@edenhill Incorporated the review comments. Please check.